qa/common/gen_qa_model_repository

```diff
@@ -210,3 +216,32 @@ if [ $? -ne 0 ]; then
     echo -e "Failed"
     exit 1
 fi
+
+# Ensemble related models
+cat >$HOST_SRCDIR/$ENSEMBLESCRIPT <<EOF
+#!/bin/bash
+set -e
+export CUDA_VISIBLE_DEVICES=$CUDA_DEVICE
+cd $SRCDIR
+python3 $SRCDIR/gen_ensemble_model_utils.py --models_dir=$ENSEMBLEDESTDIR
+chown -R $(id -u):$(id -g) $ENSEMBLEDESTDIR
+EOF
+
+chmod a+x $HOST_SRCDIR/$ENSEMBLESCRIPT
+if [ $? -ne 0 ]; then
+    echo -e "Failed: chmod"
+    exit 1
+fi
+
+# Doesn't have to use TensorFlow image, only creating model configs of util
```

Comment:
Do you need to run the container again? Can you just include this extra work in the existing TF container launch?